sdk_entrepot_gpf/workflow/action/ProcessingExecutionAction.py

```diff
@@ -232,17 +250,41 @@ def __create_processing_execution(self, datastore: Optional[str] = None) -> None
 
     def __add_tags(self) -> None:
         """Ajout des tags sur l'Upload ou la StoredData en sortie du ProcessingExecution."""
-        if "tags" not in self.definition_dict or self.definition_dict["tags"] == {} or self.__no_output:
-            # cas on a pas de tag ou vide: on ne fait rien
+        d_tags = self.definition_dict.get("tags", {})
+        # gestion des tags pour compatibility_cartes
+        if self.__mode_cartes and self.__processing_execution:
+            # mise en base de donnée livrée (vecteur)
+            if self.__processing_execution.id == Config().get_str("compatibility_cartes", "id_mise_en_base"):
```

Comment:
@LudivineSchlegel si je me trompe pas, on veut pas récupérer ce qu'il y a dans la config mais la valeur associée au tag de l’entité en entrée de la processing execution, non ?  je parle de la partie droite du if.

Reply:
`Config().get_str("compatibility_cartes", "id_mise_en_base"` est pour récupére l'id correspondant au traitement de mise en base 
C'est la partie gauche qui n'est pas bonne c'est pas l'id de la `processing_execution` mais du traitement
```suggestion
            if self.__processing_execution.get_store_properties().get("processing", {"_id": ""})["_id"] == Config().get_str("compatibility_cartes", "id_mise_en_base"):
```

Reply:
ok, ça veut dire que que l'utilisateur doit indiquer l'id du traitement qu'il considère comme étant la mise en base en config ?
C'est pas top mais en l'état ok, on pourra voir pour améliorer après